src/layouts/EditPage.jsx

```diff
@@ -22,7 +30,14 @@ export default class EditPage extends Component {
         withCredentials: true,
       });
       const { content, sha } = resp.data;
-      this.setState({ content, sha });
+      // split the markdown into front matter and content
+      const { configObj, articleContent } = frontMatterParser(Base64.decode(content));
+      this.setState({
+        content,
```

Comment:
Let's rename `content` (perhaps `mdBody` or `body`?) since `content` is the base64 string returned by the GitHub API

Reply:
renamed as frontMatter and mdBody